Automattic/jetpack
# VideoPress : allow block resize in Gutenberg

Fixes 1004-gh-Automattic/greenhouse

#### Changes proposed in this Pull Request:

Add resize handles around the video block for VideoPress in order to set a max-width.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* On a Jetpack site with VideoPress enabled
* Create a post or update an existing one
* Add a /video block
* Upload a video or select a video (uploaded on VideoPress)
* ✅ Resize handles should appear on the left and right side of the bloc
![Capture d’écran 2021-11-04 à 10 58 38](https://user-images.githubusercontent.com/1420594/140294355-e2eca33a-6034-44ba-ac96-1158dab45ab6.png)
* ✅ When resizing, the height should be automatically adjusted
* ✅ When resizing, you can't extend the size above 100% of the block
* Resize under 100% and save
* View your post on the frontend
* ✅ The video should be resized
* Open an inspector on the video and scroll up to the iframe's container (with class name wp-block-embed__wrapper )
* ✅ You should see a max-width in px and a margin: auto in the style attribute
![Capture d’écran 2021-11-04 à 11 02 55](https://user-images.githubusercontent.com/1420594/140294861-01ca9ef9-97d0-4d22-bb07-a452b763af53.png)
* Go back to Gutenberg
* Resize your video to the max
* Save your post and open the frontend view
*  Open an inspector on the video and scroll up to the iframe's container (with class name wp-block-embed__wrapper )
* ✅ The style attribute should no longer exist
